src/python/zquantum/core/utils_test.py

```diff
@@ -155,6 +155,16 @@ def test_list_io(self):
         self.assertListEqual(initial_list, loaded_list)
         os.remove("list.json")
 
+    def test_named_list_io(self):
+        # Given
+        initial_list = [0.1, 0.3, -0.3]
+        # When
+        save_list(initial_list, "list.json", "number")
+        loaded_list = load_list("list.json")
+        # Then
+        self.assertListEqual(initial_list, loaded_list)
```

Comment:
You're not checking if the schema value is properly saved here :)

Reply:
Okay, tried to add a check of this.

Reply:
Instead of defining your own `SCHEMA_VERSION`, you should do:
`from .utils import SCHEMA_VERSION` :)